spec/verb_list_comparisor_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'verb_list_comparisor'
+
+describe VerbListComparisor do
+  let(:csv_path) { 'test-verb-list.csv' }
+  subject(:verb_list_comparisor) { VerbListComparisor.new(csv_path) }
+
+  describe '#compare_databases' do
+    context 'when logged in' do
+      it 'should return a comparion of the verb for each database' do
```

Comment:
this is a fine start, but try to make sure you're specifying each thing that may change independently. this result format seems awfully specific and you may want to make some additional assertions about that format so that you have more documentation to follow in the future when the format rules change, or if nothing else, more clear documentation about what rules the format follows even if it never does change.